pyreisejl/utility/launchers.py

```diff
@@ -109,35 +148,23 @@ def launch_scenario(self, execute_dir=None, threads=None, solver_kwargs=None):
             inputfolder=self.input_dir,
             outputfolder=self.execute_dir,
             optimizer_factory=GLPK.Optimizer,
+            solver_kwargs=self.solver_kwargs,
+            num_segments=self.num_segments,
         )
         end = time()
 
-        runtime = round(end - start)
-        hours, minutes, seconds = sec2hms(runtime)
-        print(f"Run time: {hours}:{minutes:02d}:{seconds:02d}")
-
-        return runtime
+        return self.parse_and_return_runtime(start, end)
 
 
 class GurobiLauncher(Launcher):
-    def launch_scenario(self, execute_dir=None, threads=None, solver_kwargs=None):
+    def launch_scenario(self):
         """Launches the scenario.
 
-        :param None/str execute_dir: directory for execute data. None defaults to an
-            execute folder that will be created in the input directory
-        :param None/int threads: number of threads to use.
-        :param None/dict solver_kwargs: keyword arguments to pass to solver (if any).
         :return: (*int*) runtime of scenario in seconds
         """
-        self.execute_dir = execute_dir
-        self.threads = threads
         self._print_settings()
-        # Import these within function because there is a lengthy compilation step
-        from julia.api import Julia
 
-        Julia(compiled_modules=False)
-        from julia import Gurobi  # noqa: F401
-        from julia import REISE
+        Gurobi, REISE = self.init_julia(imports=["Gurobi", "REISE"])
```

Comment:
Since we don't use the gurobi import directly, you could do `_, REISE = ...`

Reply:
Done.